.ocamlformat-ignore

```diff
@@ -1 +0,0 @@
-vendor/**
```

Comment:
This file seems necessary, I'll add it back. The new one in vendor do not seem to work.